src/Solr/QueryBuilder/Update.php

```diff
@@ -14,14 +14,16 @@
  */
 class Update extends AbstractQueryBuilder
 {
+    private const STREAM_MAX_MEMORY = 50 * 1048576; // 50 Mb, default is 2 Mb
+
     private $stream = null;
 
     /**
      * Initialize the update request.
      */
     public function __construct()
     {
-        $this->stream = fopen('php://temp', 'rw');
+        $this->stream = fopen(sprintf('php://temp/maxmemory:%s', self::STREAM_MAX_MEMORY), 'rw');
```

Comment:
`%d` ? Of is het niet een integer die hier in komt?

Reply:
Het is een integer, maar hij mag naar een string worden geparsed. Het is niet zo dat ik een variabele heb die ik naar integer wil parsen.